.github/workflows/verify_generated_files.yml

```diff
@@ -15,28 +15,6 @@ on:
       - '**go.mod'
       - '**go.sum'
 jobs:
-  verify-generated-functions:
```

Comment:
What was the reason to delete this code?

Reply:
It's deleted as the entire client generation has become obsolete and been removed.